framework/db/Command.php

```diff
@@ -168,7 +168,7 @@ public function getRawSql()
         }
         $params = [];
         foreach ($this->params as $name => $value) {
-            if (is_string($name) && strncmp(':', $name, 1)) {
+            if (is_string($name) && strpos($name, ':') !== 0) {
```

Comment:
it was suggested by IDE inspections. after thinking about it I would revert it, because strpos is only faster for the positive case but not for the negative case, while strncmp is always the same.